packages/namekit-react/src/components/Checkbox.tsx

```diff
@@ -1,15 +1,40 @@
 import cc from "classcat";
 import React, { InputHTMLAttributes } from "react";
 
-export const Checkbox = (props: InputHTMLAttributes<HTMLInputElement>) => {
+export const Checkbox = ({
```

Comment:
@eduramme Hey can you please refine the logic for the click area / special mouse cursor makes sense if there's no text associated with the checkbox?

For example, in the Default story, there's a huge area that toggles the checkbox that shouldn't.

![CleanShot 2024-11-06 at 16 59 40](https://github.com/user-attachments/assets/5a11c9af-396b-4c17-93d1-75fc2e751be0)

Reply:
@lightwalker-eth done